test/powershell/Language/Operators/ReplaceOperator.Tests.ps1

```diff
@@ -40,7 +43,7 @@ Describe "Replace Operator" -Tags CI {
         }
 
         It "Replace operator can take a ScriptBlock in place of a substitution string" {
-            $res = "ID ABC123" -replace "\b[A-C]+", {return "X" * $args[0].Value.Length}
```

Comment:
I'd keep the `$args` test too.

Reply:
`$args` won't work. When used in the form of `something -replace xxx, { ... }`, the "Match" object is passed to `$_` of the script block, just like `ForEach/Where-Object {...}` and `collection.ForEach{...}/Where{...}`.